pkg/bosh/manifest/manifest_test.go

```diff
@@ -0,0 +1,409 @@
+package manifest_test
+
+import (
+	. "code.cloudfoundry.org/cf-operator/pkg/bosh/manifest"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	yaml "gopkg.in/yaml.v2"
+)
+
+var _ = Describe("Manifest", func() {
+	var (
+		job                 Job
+		jobY                []byte
+		instanceGroup       InstanceGroup
+		instanceGroupY      []byte
+		manifest            Manifest
+		manifestY           []byte
+		vmResource          VMResource
+		vmResourceY         []byte
+		network             Network
+		networkY            []byte
+		feature             Feature
+		featureY            []byte
+		release             Release
+		releaseY            []byte
+		stemcell            Stemcell
+		stemcellY           []byte
+		addOn               AddOn
+		addOnY              []byte
+		addOnPlacementRule  AddOnPlacementRules
+		addOnPlacementRuleY []byte
+		addOnStemcell       AddOnStemcell
+		addOnStemcellY      []byte
+		addOnJob            AddOnJob
+		addOnJobY           []byte
+		err                 error
+	)
+
+	vmResource = VMResource{CPU: 2,
+		RAM:               2048,
+		EphemeralDiskSize: 4096,
+	}
+
+	Context("when marshalling & unmarshalling vm resources schema", func() {
```

Comment:
Let's talk about go/ginkgo tests when you have time. I think it's enough to check if the struct has the right tags.

Like here: https://github.com/cloudfoundry/bosh-cli/blob/master/cmd/opts_test.go#L99-L105

Reply:
Done. I have changed the test cases to test the struct tags only.